Resources/config/forms.xml

```diff
@@ -13,7 +13,7 @@
     <services>
         <service id="fos_rest.form.extension.csrf_disable" class="%fos_rest.form.extension.csrf_disable.class%">
             <tag name="form.type_extension" alias="form" />
-            <argument type="service" id="security.context" />
+            <argument type="service" id="security.authorization_checker" />
```

Comment:
you would need a compiler pass changing this argument to ``security.context`` for older versions